2016/103.vtt

```diff
@@ -578,120 +523,112 @@ X-TIMESTAMP-MAP=MPEGTS:181083,LOCAL:00:00:00.000
-All right, so Joaquin
+All right, so Joaquin
 started developing this app
 
 00:07:25.776 --> 00:07:29.106 A:middle
-after he had applied for a
+after he had applied for a
 job at a local game developer,
 
 00:07:29.106 --> 00:07:30.426 A:middle
 and that hadn't worked out.
 
 00:07:30.666 --> 00:07:32.976 A:middle
-And he decided after
+And he decided after
 that that, like,
 
 00:07:33.466 --> 00:07:35.676 A:middle
-he could just do
+he could just do
 this on his own.
 
 00:07:35.796 --> 00:07:37.536 A:middle
-He didn't need to work
+He didn't need to work
 for no other developer.
 
 00:07:37.656 --> 00:07:41.846 A:middle
-So he decided to write his own
+So he decided to write his own
 app, get it on the App Store.
 
 00:07:42.146 --> 00:07:45.556 A:middle
-And I'd like to show
+And I'd like to show
 you this app.
 
 00:07:45.676 --> 00:07:47.546 A:middle
 And before I do, I just wanted
 
 00:07:47.546 --> 00:07:52.036 A:middle
-to say Joaquin was a 2015
+to say Joaquin was a 2015
 Scholarship Winner last year
 
 00:07:52.126 --> 00:07:53.746 A:middle
-at WWDC, so probably
+at WWDC, so probably
 learned a lot
 
 00:07:53.746 --> 00:07:56.646 A:middle
-about what you used
+about what you used
 in the app last year.
 
 00:07:56.646 --> 00:08:00.726 A:middle
-And he's also super active in
-the local developer community
-
-
-WEBVTT
-X-TIMESTAMP-MAP=MPEGTS:181083,LOCAL:00:00:00.000
-
-00:07:56.646 --> 00:08:00.726 A:middle
-And he's also super active in
+And he's also super active in
 the local developer community
 
 00:08:00.726 --> 00:08:02.146 A:middle
-in many of the meetups
+in many of the meetups
 in his hometown.
 
 00:08:02.146 --> 00:08:07.156 A:middle
-All right, so I'm just going to
+All right, so I'm just going to
 show you this on this screen.
 
 00:08:07.156 --> 00:08:09.616 A:middle
-So what you're seeing here
+So what you're seeing here
 is Linum is a puzzle game
 
 00:08:09.796 --> 00:08:12.036 A:middle
-where the goal is
+where the goal is
 to complete levels
 
 00:08:12.036 --> 00:08:13.606 A:middle
 in the fewest moves possible.
 
 00:08:13.606 --> 00:08:17.566 A:middle
-And what you're trying to do is
+And what you're trying to do is
 take these line nodes and lines
 
 00:08:17.566 --> 00:08:21.086 A:middle
-and pivot them around to connect
+and pivot them around to connect
 with nodes to solve the puzzles.
 
 00:08:21.556 --> 00:08:23.396 A:middle
-And as the number
+And as the number
 of nodes increases,
 
 00:08:23.696 --> 00:08:26.246 A:middle
-complex puzzles challenge
+complex puzzles challenge
 even the best players.
 
 00:08:26.246 --> 00:08:30.346 A:middle
-And the beauty of Linum
+And the beauty of Linum
 is its simplicity.
 
-00:08:30.346 --> 00:08:33.035 A:middle
-It's got a clean UI and
+00:08:30.346 --> 00:08:33.036 A:middle
+It's got a clean UI and
 a simple set of rules.
 
-00:08:33.035 --> 00:08:35.736 A:middle
-And with a smart
+00:08:33.036 --> 00:08:35.736 A:middle
+And with a smart
 implementation of touch
```

Comment:
There are two ways to fetch English VTT files. The method of changing the file extension to VTT of the standard definition video download URL occasionally results in one-millisecond timing differences from the captions resulting from HLS VTT playlist file concatenation. Fetching the monolithic files is much faster, and the monolithic files occasionally include extra captions (`[Applause]`, `[Music]`, etc) and other improvements not found in the HLS-extracted captions. The tool prefers this download method for these reasons, though it supports your choice of method. If the tool is to be the recommended means of obtaining the transcripts in the future, we should make these changes now in order to more easily recognize content improvements made by Apple in the future.